model/src/main/kotlin/org/digma/intellij/plugin/model/rest/insights/EndpointInsight.kt

```diff
@@ -2,4 +2,5 @@ package org.digma.intellij.plugin.model.rest.insights
 
 interface EndpointInsight : CodeObjectInsight {
     var route: String
+    var endpointSpan: String
```

Comment:
is it span name? or span code object id?

Reply:
this is spanName like this:
HTTP POST Transfer/TransferFunds

I named it 'endpointSpan' to be consistent with VSCode-plugin implementation